Resources/Prototypes/Entities/Structures/Power/substation.yml

```diff
@@ -9,15 +10,14 @@
   components:
   - type: Sprite # TODO: add sprite for maintenance panel open
     sprite: Structures/Power/substation.rsi
-    netsync: false
     layers:
     - state: substation
     - state: screen
       shader: unshaded
     - state: full
       shader: unshaded
   - type: Battery
-    maxCharge: 1000
+    maxCharge: 4000000
```

Comment:
Should this be significantly higher than powernetworkbattery

Reply:
This is for the base entity. Not sure exactly why it's set so low when the one we actually use, SalternSubstation just replaces the component anyway with the 4MW max charge.

Construction graphs use the base entity so that they spawn not fully charged so otherwise we need a separate entity for that if we don't want to touch the base.